apis/python/src/tiledb/vector_search/module.cc

```diff
@@ -14,6 +14,9 @@ using Ctx = tiledb::Context;
 bool global_debug = true;
 double global_time_of_interest;
 
+bool enable_stats = false;
+FILE* stats_file = stdout;
```

Comment:
I tried commenting them out and did not get linker errors because the `extern` symbols were not found. Does anyone know why?